apps/channels/tests/test_facebook_integration.py

```diff
@@ -60,6 +60,7 @@ def test_parse_messages(self, message, message_type):
             (twilio_messages.Messenger.audio_message(), "audio"),
         ],
     )
+    @patch("apps.service_providers.messaging_service.settings.AWS_ACCESS_KEY_ID", "123")
```

Comment:
This fixes that failing test. I couldn't get [this method](https://stackoverflow.com/questions/31582750/python-mock-patch-os-environ-and-return-value) to work, so resorting to this alternative.

Reply:
Did you try [@override_settings](https://docs.djangoproject.com/en/5.0/topics/testing/tools/#django.test.override_settings)?

Reply:
I did not (I might have searched for the wrong thing!), but I did [use it here](https://github.com/dimagi/open-chat-studio/pull/350/commits/58b16d59073b5bd30121c4e2fb66bce6cec9c578) and it works! Thanks Simon!